manual_qa_tests/initialize.py

```diff
@@ -18,6 +20,8 @@
 # when adding new deployments, keep the order of `fail`, `warning`, and `succeed`
 # such that the UI would approximately show these workflows in reverse order.
 WORKFLOW_PKGS = [
+    dag_layout_test,
+    march_madness_dag_layout_test,
     check_status_test,
     fail_bad_check,
```

Comment:
I know I also forgot to follow the ordering here, but would appreciate if you could help sorting them in fail - warning - success order :)

Reply:
done!